security/guard_authentication.rst

```diff
@@ -164,14 +164,18 @@ This requires you to implement six methods::
     class TokenAuthenticator extends AbstractGuardAuthenticator
     {
         /**
-         * Called on every request. Return whatever credentials you want,
-         * or null to stop authentication.
+         * Called on every request. Return whatever credentials you want to
+         * be passed to getUser().  Returning null will cause this authenticator
+         * to be skipped.
          */
         public function getCredentials(Request $request)
         {
             if (!$token = $request->headers->get('X-AUTH-TOKEN')) {
-                // no token? Return null and no other methods will be called
-                return;
+                // No token?
+                // Throwing an exception will cause authentication
+                //   to fail and prevent other authenticators from
```

Comment:
Can you please reduce the number of spaces here so that all comments are aligned?